### ISSUE TITLE
Open hyper here

### ISSUE THREAD

**Issue by Hongarc** [hharnisc/hypercwd#68]
I think when I choose `Open hyper here` in context, don't need open `initialWorkingDirectory`, Can you check it to disable when I choose `Open hyper here`  in my directory or create option to force use `initialWorkingDirectory`.

Thank you.

**Comment by jhynzar** [comment]
I think you can set it to an empty string ('').

**Comment by hong4rc** [comment]
Yes, I use that way, but that way mean don't use `hypercwd`, I want if I don't choose a folder, `Hyper` will open a new terminal same folder.
